doc-content/enterprise-only/cases/case-management-cancel-or-destroy-case-proc.adoc

```diff
@@ -1,13 +1,15 @@
 [id='case-management-cancel-or-destroy-case-proc-{context}']
 = Canceling or destroying a case
 
-Cases can be canceled if they are no longer required and do not require any case work to be performed. Cases that are canceled can be reopened later with the same case instance ID and case file data. In some cases, you might want to permanently destroy a case so that it cannot be reopened. 
+*<MH>Cristiano, can this be done using the showcase app?</MH>*
```

Comment:
It is. Same as the instructions for Close. The Cancel button is now the one hidden in the three dots button for both Case List and Case details.

Reply:
Got it. Thanks